src/components/Header.js

```diff
@@ -58,8 +84,14 @@ export default class Header extends Component {
             </div>
           </div>
 			    <div className="header__summary-dashboard">
+            {
+              siteID &&
+              <div className="header__site-logo">
+                <img src={process.env.PUBLIC_URL + "/assets/images/" + siteID + "_logo.png"} alt="site logo" onLoad={this.handleImageLoaded} onError={this.handleImageLoadError}/>
```

Comment:
If you expect there to be a lot of assets, it may make more sense for each site to have its own subdirectory of overridden assets (CSS, images) (credit to @pbugni )

Reply:
@ivan-c thank you, Ivan! I made the fix. Will you review my changes again when you get a chance?